azerothcore/azerothcore-wotlk
# fix(DB/Loot): fixes the drop rate for Princess Theradras

## Changes Proposed:
-  Sets the drop rate in accordance with given values in closed issue.

## Issues Addressed:

- Closes #7899
- Closes https://github.com/chromiecraft/chromiecraft/issues/1735

## SOURCE:


## Tests Performed:

- As below


## How to Test the Changes:


1. Check that the correct number of rows changes
2. Verify drop rate in DB
    * Should be set to the correct value for all the expected rows
    * Should have a sum of 100

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
